modules.json

```diff
@@ -185,5 +185,15 @@
       "RedBeardLab"
     ],
     "stars": 613
+  },
+  {
+    "name": "lqrm",
+    "license": "BSD",
+    "repository": "https://github.com/halaei/lqrm",
+    "description": "A Laravel compatible queue driver for Redis that supports reliable blocking pop from FIFO and scheduled queues.",
+    "authors": [
+      "Hamid Alaei Varnosfaderani"
```

Comment:
This needs to be a valid Twitter (not GitHub) handle - it is ok to leave empty if you don't have one.